README.md

```diff
@@ -76,28 +76,21 @@ All workflows are using the tagging convention listed below. It is subsequently
 
 ### Get the tools
 
-Download `docker-compose.yml` and `mailserver.env`
+Since Docker Mailserver `v10.2.0`, you can run the commands you previously executed with `./setup.sh ...` directly via `docker exec <CONTAINER> setup ...` if there already is a container running. The script `setup.sh` is here for convenience, and if you're using Docker or Docker Compose, it is recommended to use `setup.sh` if you're new to Docker Mailserver.
 
 ``` BASH
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/docker-compose.yml
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/mailserver.env
-```
-
-and the `setup.sh` **in the correct version**
-
-``` BASH
-# if you're using :edge as the image tag
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master/setup.sh
-# if you're using :latest (= :10.1.1) as the image tag
-wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/v10.1.2/setup.sh
+DMS_GITHUB_URL='https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/master'
+wget "${DMS_GITHUB_URL}/docker-compose.yml"
+wget "${DMS_GITHUB_URL}/mailserver.env"
+wget "${DMS_GITHUB_URL}/setup.sh"
 
 chmod a+x ./setup.sh
-
-# and make yourself familiar with the script
 ./setup.sh help
 ```
 
-**Make sure to get the `setup.sh` that comes with the release you're using**. Look up the release and the git commit on which this release is based upon by selecting the appropriate tag on GitHub. This can done with the "Switch branches/tags" button on GitHub, choosing the right tag. This is done in order to rule out possible inconsistencies between versions.
+Running `./setup.sh help` will also pull the `:latest` image (which is the latest stable release), if this has not been done beforehand.
```

Comment:
Is this accurate?

Looking at the file (with my stale shell skills) any command used seems to call the container method which will use an existing running container (including`help`)?

It's only when a container isn't running that it pulls the latest `:latest` tag from a registry?

The last part "if this has not been done beforehand", would it not remove the container after satisfying the command in this case, so repeat the same flow by checking the registry again to pull?

Reply:
You're quite right.

> The last part "if this has not been done beforehand", would it not remove the container after satisfying the command in this case, so repeat the same flow by checking the registry again to pull?

Could you rephrase this question for me. Not I can follow here.

Reply:
> Accurate.

Ok, so `./setup.sh help` may not actually pull `:latest` if it detects an existing DMS container running. That should be rephrased then.

> Could you rephrase this question for me. Not sure I can follow here.

If no container is running, and I run `./setup.sh help`, then a docker registry is checked for newer `:latest` tag to pull, container is created to run the command, and then exits.

Calling it again will play out the same sequence, unless the digest of `:latest` on the registry is updated.

I guess "if this has not been done beforehand" is hinting that the pull download could potentially be skipped? (`:latest` has been not updated since the last registry check).

---

Could just be me, I'm not fussed if you think the text is fine as-is :)

Two attempts at re-phrasing it:

```suggestion
If no `docker-mailserver` container is running, any `./setup.sh` command will check online for the `:latest` image tag (the current stable release), performing a `pull` if necessary followed by running the command in a temporary container. 
```

or

```suggestion
`./setup.sh help` (_or any supported command_) with no active `docker-mailserver` container will _pull_ the latest stable release (_`:latest` image tag_) before running the command.
```

Reply:
Went with the former :)